src/core/managers/internal/ToastManager/helpers/tests/createToastsFromTransactions.test.ts

```diff
@@ -0,0 +1,211 @@
+import { testAddress } from '__mocks__/accountConfig';
+import {
+  getIsTransactionPending,
```

Comment:
in general I find that we should implement such complex tests at the end when things are stable but we can leave it if you say it's OK to have it now

Reply:
They are very useful for the toasts logic